core/benches/poh.rs

```diff
@@ -61,3 +65,33 @@ fn bench_poh_lock_time_per_batch(bencher: &mut Bencher) {
         poh.hash(NUM_HASHES_PER_BATCH);
     })
 }
+
+#[bench]
```

Comment:
This isn't really a bench in the true sense. I'd just skip it. We've got this in setup.sh anyway. If you decide to merge this anyway, make sure it's ignored. No use running it every time in CI.